src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -734,8 +735,8 @@ bool OutputStateMachineEngine::ActionOscDispatch(const wchar_t /*wch*/,
     std::wstring params;
     std::wstring uri;
     bool queryClipboard = false;
-    size_t tableIndex = 0;
-    DWORD color = 0;
+    std::vector<size_t> tableIndexes;

```

Comment:
heads-up: this function will conflict with @j4james' work in #7799

Reply:
I haven't touched the OSC side of things, so I hoped there wouldn't be any conflict, but I'm happy to rebase if necessary after this merges.

Reply:
Thanks @j4james . I think it might be better to merge #7799 first since that is obviously a larger PR. I'm OK with rebasing if necessary.